src/extension.ts

```diff
@@ -9,14 +9,19 @@ import * as VSCode from "vscode";
 import Program from "./Program";
 import ResourceManager from "./Properties/ResourceManager";
 import Settings from "./Properties/Settings";
+import { getMarkdownExtensionContributions } from "./System/Drawing/MarkdownExtensions";
 import Exception from "./System/Exception";
 import MarkdownFileNotFoundException from "./System/MarkdownFileNotFoundException";
 
+var markdown;
+
 // this method is called when your extension is activated
 // your extension is activated the very first time the command is executed
 export async function activate(context: VSCode.ExtensionContext)
 {
+    const contributions = getMarkdownExtensionContributions(context);

```

Comment:
Oh yeah... Also I don't think these variables need to be `const`.
You could just declare them with the `let` keyword since they will presist... let's see if the whole thing breaks when the `let`-keyword is used 😂